src/utils/tape-ava-helpers.js

```diff
@@ -115,6 +115,49 @@ export function rewriteAssertionsAndTestArgument(j, path) {
 }
 
 /**
+ * Rewrite test callback to be able to destructure its argument
+ *
+ * test(({ok}) => {ok()}) to test(t => {ok()})
+ */
+export function rewriteDestructuredTArgument(fileInfo, j, ast, testFunctionName) {
+    const rewriteOneArgument = p => {
+        // The last arg is the test callback
+        const lastArg = p.value.arguments[p.value.arguments.length - 1];
+        if (lastArg && lastArg.params && lastArg.params[0]
```

Comment:
This might make it a bit more readable:
```javascript
const lastArgParam = lastArg && lastArg.params && lastArg.params[0];
if (lastArgParam.type === 'ObjectPattern') {
    const keys = lastArgParam.properties.map(prop => prop.key.name);
```